### ISSUE TITLE
Day-ahead vs 2Day_ahead?

### ISSUE THREAD

**Issue by williamhobbs** [PERFORM-Forecasts/documentation#20]
[Here](https://github.com/PERFORM-Forecasts/documentation#Forecasts) you describe day-ahead, intra-day, and intra-hour forecasts, but it seems like the "day-ahead" forecast may be represented in the dataset at `Day-ahead` and `2Day_ahead` (e.g., for solar in ERCOT, [2Day_ahead](https://data.openei.org/s3_viewer?bucket=arpa-e-perform&prefix=ERCOT%2F2018%2FSolar%2FForecasts%2F2Day_ahead%2F) and [Day-ahead](https://data.openei.org/s3_viewer?bucket=arpa-e-perform&prefix=ERCOT%2F2018%2FSolar%2FForecasts%2FDay-ahead%2F). 

This would make sense, as this,

> Day-ahead forecasts are generated with a 11-hour-ahead lead time, a 48-hour horizon, an hourly resolution, and a daily update rate

results in overlapping time series, where there are more than one forecasted value each forecast time.

Can you clarify this? Thanks!

For context, this came up when I was looking at this question, https://stats.stackexchange.com/questions/586078/what-does-this-statement-about-a-forecasting-lead-time-mean/.

**Comment by bsergi** [comment]
You are correct about the day-ahead being represented by both the `Day-ahead` and `2Day_ahead` datasets, with the latter represented the forecast 2 days out. @fzhang-WZ or @fengcong1992 might be able to provide more details if needed.
